### DISCUSSION TITLE
What's new on v2.x

### DISCUSSION THREAD

**Discussion by ftrescue** [datarhei/restreamer#515] [General]
Just had a note saying there is an update can i ask what is new before i upgrade thanks

**Comment by svenerbeck** [comment]
Hi @ftrescue  [electronic high5]

That's a valid question. We've been working diligently to improve our datarhei Core technology for upcoming updates. We're happy to report that we've significantly reduced HLS latency. Our latest tests have shown just 5 seconds of latency - significantly improving over previous versions. In addition, thereIn addition, there are more setting options for this in the GUI for pros to control the latency. And if you need even lower latency, SRT is an excellent option with just 0.3 seconds of delay. We're committed to providing the best experience for our users, and we can't wait for you to see these improvements in action.

Changelog here: [https://github.com/datarhei/restreamer/blob/2.x/CHANGELOG.md](https://github.com/datarhei/restreamer/blob/2.x/CHANGELOG.md)

Cheerio 
Sven
